src/asm2wasm.h

```diff
@@ -1696,6 +1696,7 @@ void Asm2WasmBuilder::processAsm(Ref ast) {
     passRunner.add("merge-blocks");
     passRunner.add("optimize-instructions");
     passRunner.add("post-emscripten");
+    passRunner.add("ImportsToIndirectCalls");
```

Comment:
You probably don't want to add this here.

Reply:
Ok, can you suggest a better place to add this?

Reply:
This was a mistake. Will remove it.